src/client/apps/edit/components/content/article_layouts/test/news.test.tsx

```diff
@@ -1,13 +1,13 @@
-import React from "react"
-import configureStore from "redux-mock-store"
+import { NewsByline } from "@artsy/reaction/dist/Components/Publishing/Byline/NewsByline"
 import { NewsArticle } from "@artsy/reaction/dist/Components/Publishing/Fixtures/Articles"
 import { NewsHeadline } from "@artsy/reaction/dist/Components/Publishing/News/NewsHeadline"
-import { NewsByline } from "@artsy/reaction/dist/Components/Publishing/Byline/NewsByline"
-import { Provider } from "react-redux"
+import { EditSourceControls } from "client/apps/edit/components/content/sections/news/EditSourceControls"
```

Comment:
So when converting to tsx the entire path is require instead of `"../../sections/news/EditSourceControls"`?

Reply:
Yeah we are moving towards using absolute paths for everything, makes moving and tracing the source of files much easier. Same as we are doing in Reaction.

Reply:
Got it!